korge/src/common/korlibs/korge/view/SolidTriangle.kt

```diff
@@ -21,23 +21,36 @@ class SolidTriangle(
 ) {
   var p1: Point = p1
     set(value) {
+      updateTriangle()
       updatedVertices()
       field = value
     }
   var p2: Point = p2
     set(value) {
+      updateTriangle()
       updatedVertices()
       field = value
     }
   var p3: Point = p3
     set(value) {
+      updateTriangle()
       updatedVertices()
```

Comment:
These calls to `updatedVertices` might be moved to updateTriangle?

Reply:
You're right. Done.